### PR TITLE
Add more informative error message about duplicate EP modifiers

### PR DESCRIPTION
**Pull request by shanear**
Also update travis ruby version to match ours.

connects #1281

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 language: ruby
 dist: wily
 rvm:
-  - 2.3.0
+  - 2.2.4
```
Comment (joofsh) [inline .travis.yml]: 🙌

File: app/controllers/establish_claims_controller.rb [hunk 2]
```diff
@@ -14,6 +14,9 @@ def perform
       Dispatch.new(claim: establish_claim_params, task: task).establish_claim!
     end
     render json: {}
+
+  rescue Dispatch::EndProductAlreadyExistsError => e
+    render json: { error_code: "duplicate_ep" }, status: 500
```
Comment (joofsh) [inline app/controllers/establish_claims_controller.rb]: 500 wouldn't be semantically valid (this isn't a caseflow server error). I think `422` makes the most sense?

File: app/services/dispatch.rb [hunk 3]
```diff
@@ -67,6 +68,9 @@ def establish_claim!
                                                      appeal: task.appeal)
 
     task.review!(outgoing_reference_id: end_product.claim_id)
+
+  rescue VBMS::HTTPError => error
+    raise parse_vbms_error(error)
```
Comment (joofsh) [inline app/services/dispatch.rb]: 👍

File: app/controllers/establish_claims_controller.rb (hunk 2 above)
Comment (shanear) [inline app/controllers/establish_claims_controller.rb]: 👍

File: app/services/dispatch.rb (hunk 3 above)
Comment (shanear) [inline app/services/dispatch.rb]: watching too many refactoring tech talks 💻

**Pull request merged**